src/app/screens/Publishers/noPublishers.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { render, screen } from "@testing-library/react";
```

Comment:
Are there two test files here now? The other is index.test.tsx?

Reply:
Yes, looks like it:
https://github.com/getAlby/lightning-browser-extension/tree/e316da4de115eb542160170a836e0f204a069338/src/app/screens/Publishers

@rolznz I think you could just move that test into the test-index file. Or did you have any issues with that?

Reply:
@escapedcat  @qqqzhch  I had some trouble with the mocks, but I found a solution. I have removed the test file I added and moved the test I made into the index test file